packages/components/text/src/Text.tsx

```diff
@@ -119,14 +125,30 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
 
   // return a continuation function that allows this text to be compressed
   return (extra: TextProps, children: React.ReactNode) => {
+    // [TODO(#2268): Remove once RN Core properly supports Dynamic Type scaling
+    interface ScaleStyleAdjustmentProps {
+      fontSize?: number;
+      lineHeight?: number;
+    }
```

Comment:
I'd move the type definition out of here into either the top or a types file if possible. Or better yet, re-use an existing type?

Reply:
This was done as an initial thing because token typing isn't necessarily obvious upon first glance. After digging a bit deeper, it looks like `TextTokens` works just fine here.